stock_packaging_usability/readme/USAGE.rst

```diff
@@ -0,0 +1,7 @@
+This module adds 2 buttons in the *Operations* lines of a picking:
+
+* *Put in current pack*
+
+* *Put in new pack*
+
+It is a complement of the button *Put in Pack* which is available on the picking. With these 2 new buttons, you are not obliged to set the *Done quantity* package by package.
```

Comment:
This doesn't seem like an appropriate usage section. AFAIK it should contain a step by step guide on how to exercise this module's features. Check out [the instructions in the template](https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/USAGE.rst).

This belongs more to a description or context. But, if I read these instructions, I don't know where to go or what to do to test the module on runboat.

Reply:
I agree with your remark and I made the change